units/patch_intent.go

```diff
@@ -493,11 +493,23 @@ func (j *patchIntentProcessor) processTriggerAliases(ctx context.Context, p *pat
 	}
 
 	for _, intent := range triggerIntents {
-		if err := j.env.RemoteQueue().Put(ctx, NewPatchIntentProcessor(mgobson.ObjectIdHex(intent.ID()), intent)); err != nil {
-			return errors.Wrap(err, "problem enqueueing child patch processing")
+		triggerIntent, ok := intent.(*patch.TriggerIntent)
+		if !ok {
+			return errors.Errorf("intent '%s' didn't not have expected type '%T'", intent.ID(), intent)
 		}
-	}
 
+		if triggerIntent.ParentStatus == "" {
```

Comment:
maybe we should leave a comment why we want to process it right away in this case